src/locales/en_US.ts

```diff
@@ -163,7 +163,8 @@ const strings = {
   fragment_transaction_list_confirmation_progress: '%s of %s Confirmations',
   fragment_transaction_list_transaction: 'Transactions',
   transaction_list_buy_crypto_message: 'Buy %s',
-  transaction_list_no_tx_yet: 'No transactions yet!',
+  transaction_list_no_tx_yet: 'No transactions yet.',
+  transaction_list_no_tx_support_yet: 'Transaction history is not yet supported.',
```

Comment:
I say we avoid using ending punctuation for these one-off statements scattered across the app. Ending punctuation makes sense when you need to separate things but strings like this, to me, look better without.